2021q3/boot-performance.adoc

```diff
@@ -0,0 +1,30 @@
+== Boot Performance Improvements
+
+Links
+link:https://wiki.freebsd.org/BootTime[Wiki page]
+link:https://www.daemonology.net/blog/2021-08-12-EC2-boot-time-benchmarking.html[OS boot time comparison]
+
+Contact: Colin Percival <cperciva@FreeBSD.org>
+
+Colin Percival is coordinating an effort to speed up the FreeBSD boot process.
+For benchmarking purposes, he is using an EC2 c5.xlarge instance as a reference
+platform and is measuring the time between when the virtual machine enters the
+EC2 "running" state and when it is possible to SSH into the instance.
+
+This work started in 2017 and led to a conference presentation, "Profiling
+the FreeBSD kernel boot", and quickly yielded roughly 4850 ms of improvements
+(starting from a baseline of about 30 seconds).
+
+Since June, another roughly 9790 ms of time has been shaved off the boot
+process, taking it down to approximately 15 seconds.  There is still more work
+to be done; in particular, while the loader and kernel have been profiled, the
+TSLOG system Colin is using does not currently support userland profiling.
+
+Issues are listed on the wiki page as they are identified; the wiki page also
+has instructions for performing profiling.  Users are encouraged to profile
+the boot process on their own systems, in case they experience delays which
+don't show up on the system Colin is using for testing.
+
+This work is being supported by the Colin's FreeBSD/EC2 Patreon.
```

Comment:
s/by the /by/

I'd also s/being supported/supported/, but again, taste and dialect differences.

Reply:
Thanks!